tests/generation/FileGlobs.py

```diff
@@ -29,7 +29,7 @@ def test_get_project_files(self):
         open(os.path.join("ignore_dir", "src.c"), "w").close()
         open(os.path.join("ignore_dir", "src.js"), "w").close()
 
-        with suppress_stdout(), simulate_console_inputs("**.c", "ignore_dir/**"):
+        with suppress_stdout(), simulate_console_inputs("ignore_dir/**"):
```

Comment:
i cant find simulate_console_inputs anywhere, why are you using 1 param instead of one now?

and you also have ``.c`` files now, why not take them too?